widgets/testimonial/tpl/default.php

```diff
@@ -17,12 +17,22 @@
 				<?php endif; ?>
 
 				<div class="sow-testimonial-user">
+					<?php if( $sow_user_image = $this->testimonial_user_image($testimonial, $design) ){ ?>
 					<div class="sow-image-wrapper">
-						<?php echo $this->testimonial_user_image($testimonial, $design) ?>
+						<?php echo $sow_user_image ?>
 					</div>
+					<?php } ?>
 					<div class="sow-text">
 						<strong><?php echo esc_html( $testimonial['name'] ) ?></strong>
-						<a><?php echo esc_html( $testimonial['location'] ) ?></a>
+						<?php if( !empty( $testimonial['location'] ) ){ ?>
+						<?php   if( empty( $testimonial['url'] ) ){ ?>
+						<?php echo esc_html( $testimonial['location'] ) ?>
+						<?php   }elseif( $testimonial['new_window'] ){ ?>
+						<a href="<?php echo $testimonial['url']; ?>" target="_blank"><?php echo esc_html( $testimonial['location'] ) ?></a>
```

Comment:
The `url` field should be sanitized before being output to the front end, as described here: https://codex.wordpress.org/Data_Validation#URLs

We have a wrapper function for it called `sow_esc_url` which you can use here and in the line below.